<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>4800</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/map.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>384</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/game_name.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="car">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>237</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/car.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start_game">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/src/start_btn.png</normaloff>:/src/start_btn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>250</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_distance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>300</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>行駛距離: 0公尺</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_clock">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="block_01">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block_02">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block_05">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block_03">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block_06">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>390</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block_04">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item2.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
